source/message/OP_MSG.rst

```diff
@@ -206,6 +215,40 @@ connection. All other errors during processing will be silently dropped, and
 will not result in the connection being closed.
 
 
+moreToCome On Responses
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When the ``moreToCome`` flag is set on a response it signals to the recipient
+that the sender will send additional responses on the connection. The recipient
+MUST continue to read responses until it reads a response with the ``moreToCome``
+flag not set, and MUST NOT send any more requests on this connection until
+it reads a response with the ``moreToCome`` flag not set. The client MUST
+either consume all messages with ``moreToCome`` flag set or close the connection.
+
+When the server sends responses with ``moreToCome`` flag set, it behaves as if
+the client sent a ``getMore`` request for the returned cursor for each of the
+responses. Each reply will have a unique ``messageId``, and the ``responseTo``
+field of every follow-up message will be the `messageId`` of the previous
+reply. The reply will also include the normal command response body to the
+implicit ``getMore``.
+
+The client MUST be prepared to receive a response without ``moreToCome`` set
```

Comment:
Is there any way to force this to happen so that driver can test it?

Reply:
Sure, by using a mock server.

Reply:
Sure, but I meant a real server

Reply:
I am not aware of a mechanism for a driver to force the server exhausting or not exhausting.